Formula/dnstwist.rb

```diff
@@ -83,9 +83,6 @@ def install
     venv = virtualenv_create(libexec, "python3")
     venv.pip_install resources
 
-    # Replace shebang with virtualenv python
-    inreplace "dnstwist.py", "#!/usr/bin/env python", "#!#{libexec}/bin/python3"
```

Comment:
In `dnstwist.py` file, now the shebang defaults to python3, so if we still do string replacement it would ended up with python33 in the build failure.